tools/qmctl/README.md

```diff
@@ -0,0 +1,43 @@
+# QMCTL
+
+QMCTL is a command-line tool for managing QM containers using Podman. It provides a simple interface to inspect container internals, run commands, and monitor system-level resources.
```

Comment:
see the QMCTL.__init__ the default is QM no other container in the host/

Consider this one 
QMCTL is a command-line tool for managing QM partitioned container.
or containers in general? 

Consider this description instead 
QMCTL offers a simple interface to inspect QM container namespaces, monitor cgroups and systemd resources, and run commands inside QM containers.